ERCS/erc-7777.md

```diff
@@ -0,0 +1,137 @@
+---
+eip: 7777
```

Comment:
```suggestion
eip: 7726
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.